tox.ini

```diff
@@ -45,3 +45,4 @@ commands=
     flake8 {toxinidir}/<MODULE_NAME> {toxinidir}/tests
     isort --recursive --check-only --diff {toxinidir}/<MODULE_NAME> {toxinidir}/tests
     pydocstyle --explain {toxinidir}/<MODULE_NAME> {toxinidir}/tests
+    black {toxinidir}/<MODULE_NAME> {toxinidir}/tests --check
```

Comment:
Super nitpicky, but I think it would be nicer to put `--check` before the sources, even if it works this way, just to follow the standard set by the help docs:
```sh
$ black -h
Usage: black [OPTIONS] SRC ...
```